airflow/www/views.py

```diff
@@ -33,7 +33,6 @@
 import sqlalchemy as sqla
```

Comment:
Did you run some sort of autopep8 lint-fixer on this? Looks like a lot of minor line changes that are unrelated to any logic changes needed for this feature. If so, can you undo them? It will make it easier to spot logic-only changes. At some point, we might need to do an autopep8 run on all files, but we will wait for that day.